.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+#  - name: Question
+#    about: If you have a question, ask it in a discussion
+#    url: https://github.com/nativeci/issues/discussions
```

Comment:
if you want to use discussions for questions like here:
![image](https://user-images.githubusercontent.com/73652197/126047777-5a1a5631-3905-43d8-a08c-8cb73ed3e643.png)

please enable them in settings, else just let me know, and I'll remove these lines

Reply:
Wow great stuff, I have enabled the discussions